src/Request.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Saritasa\DingoApi\Utils;
+
+/**
+ * Requests helper utility. Allows to check is request secure and check request's header value.
+ */
+class Request
```

Comment:
I'd move this class to some package like saritasa/laravel-helpers, but I'm not sure, it worths to create one. Maybe move it to saritasa/laravel-middleware, and reference in this package?